apiserver/src/test/java/org/dependencytrack/util/ConfigUtilTest.java

```diff
@@ -34,6 +35,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Ignore // Dirties the JVM which doesn't play nicely with Maven Surefire reusing a JVM for all tests.
```

Comment:
Necessary to make use of `reuseForks=true` in Maven. This test being disabled is a fair tradeoff for tests running significantly faster IMO.